spec/api/v1/authentication/get_token_auth_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'token authentication' do
+  describe 'user new' do
+    it ''
+  end
+end 
```

Comment:
We should def start building out tests for this as well to figure out happy vs sad paths in the app. Not 100% critical for this PR, but let's maybe make another issue for that in the Backend To Do?